python_modules/dagster/dagster/_core/execution/plan/compute.py

```diff
@@ -146,7 +146,7 @@ def _yield_compute_results(
 ) -> Iterator[OpOutputUnion]:
     check.inst_param(step_context, "step_context", StepExecutionContext)
 
-    context = OpExecutionContext(step_context)
+    context = build_execution_context(step_context)
```

Comment:
@alangenfeld  is this the only place that needs to be updated to use `build_execution_context`? it's the only callsite updated in this [PR](https://github.com/dagster-io/dagster/pull/14814/files#diff-2a72c1ffd0fe9687dfa6d836ce449a31b356b704e855c6f75b40d581e236db72), but just want to confirm you don't know of any other code paths

Reply:
ya this is the place for the core execution flow

**direct** op / asset execution is a separate code path you will need to consider / grapple with